caikit/core/model_management/local_model_trainer.py

```diff
@@ -277,6 +295,7 @@ def train(
             save_with_id=save_with_id,
             external_training_id=external_training_id,
             use_subprocess=self._use_subprocess,
+            subprocess_start_method=self._subprocess_start_method,
```

Comment:
Do we need to be doing something different that's not just `**kwargs` here to pass along the training kwargs?

If I already had `subprocess_start_method` as a kwarg for the train method, then this would raise something like `got multiple values for argument subprocess_start_method`.
Probably not a huge deal but we're starting to put quite a few things alongside the training parameters and I'd think we should set out a good pattern in the local trainer for other implementations to reference

Reply:
Very good call. Pretty easy fix to just take `args`/`kwargs` as explicit arguments and not do `*`/`**` until the underlying training is invoked